src/app/organizations/state/collections.service.ts

```diff
@@ -53,8 +51,8 @@ export class CollectionsService {
     this.collectionsStore.setLoading(true);
     const activeId: ID = this.collectionsQuery.getActiveId();
     this.collectionsStore.remove();
-    this.organizationsService
-      .getCollectionsFromOrganization(organizationID)
+    this.openApiOrganizationsService
```

Comment:
Suggest removing `private openApiOrganizationsService: OpenApiOrganizationsService` and keeping `private organizationsService: OrganizationsService` so you don't have to update the below usages